docs/api/README.md

```diff
@@ -0,0 +1,27 @@
+# trustlines-clientlib
+
+## Index
+
+### External modules
+
+- ["Contact"](modules/_contact_.md)
```

Comment:
So that's auto-generated. Can we configure it to don't use such ugly quotes? :thinking:

Reply:
After a little bit of digging, I could not find a straightforward way to remove them unfortunately.. Have a look at https://github.com/TypeStrong/typedoc/issues/68

Reply:
Okay then we have to live with that. :man_shrugging: 
Thanks for the additional research!